### ISSUE TITLE
sql_scanner.h cannot pass the local pre-commit

### ISSUE THREAD

**Issue by AngLi-Leon** [cmu-db/peloton#599]
```
04-01-2017 15:01:12 [<module>:143] INFO : Scanning : /Users/angli/Documents/peloton/script/validators/../../src
04-01-2017 15:01:13 [validate_file:108] INFO : Invalid pattern --  free\( -- found in : /Users/angli/Documents/peloton/script/validators/../../src/include/parser/sql_scanner.h
04-01-2017 15:01:13 [validate_file:109] INFO : Line :: 	 * and can realloc() it to grow it, and should free() it to

04-01-2017 15:01:13 [<module>:147] INFO : Validation not successful
```

**Comment by AngLi-Leon** [comment]
Looks like this file is already removed from the main repo. I will rebase my repo.

**Comment by apavlo** [comment]
Already fixed in 834ccde09f458806319ed7d5e7202154d57dded7
